packages/auto-label/src/auto-label.ts

```diff
@@ -363,18 +355,47 @@ export function handler(app: Application) {
       repo,
       pull_number,
     });
-    const language = langlabler.getPRLanguage(
-      filesChanged.data,
-      config.language
-    );
-    if (language) {
-      logger.info('Labeling PR with: ' + language);
-      await context.github.issues.addLabels({
-        owner,
-        repo,
-        issue_number: pull_number,
-        labels: [language],
-      });
+
+    if (config.path && config.path.pullrequest) {
```

Comment:
Would you mind if we set comments above some of the 'if' statements just to explain the paths?

Reply:
That's good point. I'll add more comments to this PR, since the various configs are now getting hard to follow